src/controllers/Document.js

```diff
@@ -220,6 +220,19 @@ class DocumentController extends BaseController {
       .then(response => response.result);
   }
 
+  updateByQuery(index, collection, searchQuery, changes, options = {}) {
+    const request = {
+      index,
+      collection,
+      body: {...searchQuery, ...changes},
```

Comment:
This should not work because every properties is at the same level instead of being inside the `searchQuery` and `changes` properties

Reply:
?
In searchQuery you have like 
```{ query: { match: { field: "value" } } }```
and in changes
```{ changes: { field: "value2" } }``` 
so it's ok, and works

Reply:
Hm you say that i should wrap it ? query and changes properties?